src/entity/user/experiences/UserExperienceWork.ts

```diff
@@ -0,0 +1,25 @@
+import { ChildEntity, Column, OneToMany } from 'typeorm';
+import { UserExperience } from './UserExperience';
+import { UserExperienceType } from './types';
+import { EmploymentType } from '@dailydotdev/schema';
+import type { UserExperienceSkill } from './UserExperienceSkill';
+
+@ChildEntity(UserExperienceType.Work)
+export class UserExperienceWork extends UserExperience {
+  @Column({
+    type: 'integer',
+    comment: 'EmploymentType from protobuf schema',
+    default: EmploymentType.UNSPECIFIED,
```

Comment:
should be null instead. `UNSPECIFIED` is a fallback value. Kinda like `someObject?.notExist` will be `undefined`